agenta-backend/agenta_backend/resources/evaluators/evaluators.py

```diff
@@ -91,6 +91,43 @@
         },
         "description": "JSON Field Match evaluator compares specific fields within JSON (JavaScript Object Notation) data. This matching can involve finding similarities or correspondences between fields in different JSON objects.",
     },
+    {
+        "name": "JSON Diff Match",
+        "key": "auto_json_diff",
+        "direct_use": False,
+        "description": "JSON Diff evaluator compares two JSON objects to identify differences. It highlights discrepancies, additions, deletions, and modifications between the objects, providing a clear report of how they differ.",
```

Comment:
At some point we need to clean up all the copy in the docs and the app.
But now is not the time :D

Reply:
Noted. 🫡